tests/langchain/test_langchain_model_export.py

```diff
@@ -473,6 +474,40 @@ def test_langchain_agent_model_predict(return_intermediate_steps):
         )
 
 
+def test_langchain_agent_model_predict_stream():
+    langchain_agent_output = {
+        "id": "chatcmpl-123",
+        "object": "chat.completion",
+        "created": 1677652288,
+        "choices": [
+            {
+                "index": 0,
+                "finish_reason": "stop",
+                "text": f"Final Answer: {TEST_CONTENT}",
+            }
+        ],
+        "usage": {"prompt_tokens": 9, "completion_tokens": 12, "total_tokens": 21},
+    }
+    model = create_openai_llmagent()
+
+    with mlflow.start_run():
+        logged_model = mlflow.langchain.log_model(model, "langchain_model")
+    loaded_model = mlflow.pyfunc.load_model(logged_model.model_uri)
+    langchain_input = {
+        "input": "What was the high temperature in SF yesterday in Fahrenheit?"
+        "What is that number raised to the .023 power?"
+    }
```

Comment:
```suggestion
    langchain_input = {"input": "foo"}
```

to indicate that the input is not important at all in this test.